requirements.txt

```diff
@@ -5,5 +5,8 @@ pillow
 bibtexparser
 requests
 configparser
-pyvnml
+
+# For virtual env in ubuntu:
+# sudo apt-get install python3-dev
```

Comment:
Why is this suggestion here? I don't think we need dev libraries to install pyworkflow. 
The tk note seems more relevant in my opinion.

Reply:
@delarosatrevin thanks. This is just to annotate installation dependencies...but I'm happy yo move it somewhere else.

Reply:
I think numpy or scipy (can't remember which one was required python.h to be installed)

Reply:
Maybe in the REAME? I'm not sure if requirements.txt is used for that, I would prefer to keep it with the real deps. Anyway, I think we don't need python-dev (this is probably the old way to install). It is good to separate "building" from "installing/running"

Reply:
I'll move it to readme....but believe me, when creating a virtual environment and running pip install for scipion-app it complained about missing python.h (when installing numpy)

This was my case: https://askubuntu.com/questions/526304/cannot-compile-python-h-perhaps-you-need-to-install-python-devpython-devel

There might be other way, but I solved it that way.

Reply:
Hum...we need to check this...what dependency is requiring python.h, I don't think it is numpy.

Reply:
I took a look at the link..it is taking about PyPy, that is not the same as PyPI. ;)